persistence/src/main/java/de/cyface/persistence/model/TrackV6.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2023 Cyface GmbH
+ *
+ * This file is part of the Cyface SDK for Android.
+ *
+ * The Cyface SDK for Android is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Cyface SDK for Android is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the Cyface SDK for Android. If not, see <http://www.gnu.org/licenses/>.
+ */
+package de.cyface.persistence.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import androidx.annotation.NonNull;
+
+/**
+ * A {@link TrackV6} consists of {@link GeoLocationV6}s and {@link Pressure}s (data points) collected for a
+ * {@link Measurement}. Its data points are ordered by time.
+ * <p>
+ * A {@code TrackV6} begins with the first data point of each type collected after start or resume was triggered
+ * and stops with the last collected data point of each type before the next resume command is triggered or when the
+ * very last location is reached.
+ *
+ * @author Armin Schnabel
+ * @version 1.0.0
+ * @since 6.3.0
+ */
+public class TrackV6 {
```

Comment:
As `Track` only support `GeoLocation` I added this class which supports `GeoLocationV6` and `Pressure`.

This allowed me to use simpler code in `PersistenceLayer.loadAscend()` from pressure and location.altitude.

Reply:
Different solution would probably be to add an interface for GeoLocation and use that with Track. But as it should not be a long lasting branch, this is probably fine.